### DISCUSSION TITLE
Passing changes to Edit-R53ResourceRecordSet

### DISCUSSION THREAD

**Discussion by mcdonamw** [aws/aws-tools-for-powershell#341] [Q&A] [answered]
This is my first foray into AWS (Route53, specifically).  Is the only way to pass multiple changes to the Edit-R53ResourceRecordSet cmdlet via the ChangeBatch_Change parameter to specify them individually, e.g. `-changebatch_change $change1, $change2, etc`?  I thought the ChangeBatch object being a collection of changes would suffice here but it appears not?  

Is there a way to pass a dynamically created collection of changes?

**Comment by afroz429** [comment, marked as answer]
Hello,
The `ChangeBatch_Change` parameter is an array of `Amazon.Route53.Model.Change` records. You should be able to pass a collection. Can you share the full example of the command you are running if possible and the unexpected behavior you are seeing? 
```
➜ Get-Help Edit-R53ResourceRecordSet -Parameter ChangeBatch_Change

-ChangeBatch_Change <Amazon.Route53.Model.Change[]>
    Information about the changes to make to the record sets.

    Required?                    true
    Position?                    3
    Default value                None
    Accept pipeline input?       True (ByPropertyName)
    Accept wildcard characters?  false
```
